torch/fx/subgraph_rewriter.py

```diff
@@ -309,7 +309,10 @@ def get_replacement_nodes(curr_node: Node):
             replacement_nodes.append(curr_node)
 
         for ret_node in copied_returning_nodes:
-            get_replacement_nodes(ret_node)
+            if ret_node in match.placeholder_nodes:
+                replacement_nodes.append(ret_node)
```

Comment:
Not sure if I understand correctly, but is it possible that we still need to scan the args of the `ret_node` for `replacement_nodes` even if it is the placeholder of a match?

Reply:
`match.placeholder_nodes` are nodes in the target graph which correspond to placeholders (inputs) in the search pattern. This means that if `ret_node` is in `match.placeholder_nodes`, none of it's args are part of the search pattern, so none of them will be replaced.